userspace/libsinsp/examples/test.cpp

```diff
@@ -55,13 +55,16 @@ static bool ppm_sc_modifies_state = false;
 static bool ppm_sc_repair_state = false;
 static bool ppm_sc_state_remove_io_sc = false;
 static bool enable_glogger = false;
-static string engine_string = KMOD_ENGINE; /* Default for backward compatibility. */
+static string engine_string;
```

Comment:
This is initialized below when empty.
This way, we can check if another engine was already set and give an error.